### PR TITLE
Add meetings/download/copy route with just the 'copy all' button

### PR DESCRIPTION
**Pull request by piemonkey**
### Overview
Do not split up the document parts yet as this way the copy-all functionality and the presence of the page can be demoed. Makes use of the download-meeting-part component, adjusted to allow for putting on the clipboard instead of only downloading. Also includes some deprecation fixes that I added...

##### connected issues and PRs:
Part of https://binnenland.atlassian.net/browse/GN-5067

### Setup
N/A

### How to test/reproduce
Go to the export options for a meeting page. All existing (just added) functionality should work, but the 'more copy options' button should take you to the new screen. After clicking 'copy full decision' the content should be on your system clipboard.

### Challenges/uncertainties
N/A

### Checks PR readiness
- [ ] UI: works on smaller screen sizes
- [ ] UI: feedback for any loading/error states
- [ ] Check cancel/go-back flows
- [ ] Check database state correct when deleting/updating (especially regarding relationships)
- [x] changelog
- [x] npm lint
- [x] no new deprecations

### CODE REVIEW

File: translations/nl-BE.yaml [hunk 1]
```diff
@@ -801,7 +798,15 @@ download:
...
     download-html: Download HTML
     no-data-message: Geen agendapunten gevonden in zitting
+    copy-options: Bekijk kopieermogelijkehden
 
 download-meeting-part:
   downloading: Aan het downloaden
   downloaded: Gedownload
+
+copy-options:
+  heading: 'Kopieer onderdelen uit: {besluit}'
```
Comment (abeforgit) [inline translations/nl-BE.yaml:808]: the dutch version has a format param, the english one doesn't, which throws when you switch to dutch

**Review by abeforgit — CHANGES_REQUESTED** [review]
- It's arguable whether the user would want to include the template comments or not, but I think in the way the usecase is currently envisioned we should strip them out

- image
some color mismatch here (I know its nitpicking but it's likely to be the first comment from the pms :grimacing: )
- also in the above: both columns have the same title

File: translations/nl-BE.yaml (hunk 1 above)
Comment (abeforgit) [reply translations/nl-BE.yaml:808]: worded that a bit weird, but its just that it expects a param in dutch which you're not giving

Comment (piemonkey) [reply translations/nl-BE.yaml:808]: Oh, I forgot to remove the format param from the dutch when I decided to highlight the title in the same way as the heading of the download page, so had to move it out.

**Comment by piemonkey** [comment]
> * It's arguable whether the user would want to include the template comments or not, but I think in the way the usecase is currently envisioned we should strip them out

Huh, it's using the same logic as the download page, so I never actually checked if it was stripping the comments. I'll change it (which will change it for both).

>       some color mismatch here (I know its nitpicking but it's likely to be the first comment from the pms 😬 )

Huh, good spot. It turns out giving an invalid `@skin` value leaves `<AuLInk>` to use default link styling. If only we had typescript...

>     * also in the above: both columns have the same title

Oops.

File: translations/en-us.yaml [hunk 2]
```diff
@@ -797,7 +794,16 @@ download:
...
 download-meeting-part:
   downloading: Downloading
   downloaded: Downloaded
+
+copy-options:
+  heading: 'Copy parts from:'
+  disclaimer: 'The formatting of the document will depend on the application in which it is loaded.'
+  copy-all: Copy full decision
+  copying: copying
+  completed: copied
```
Comment (elpoelma) [inline translations/en-us.yaml:809]: The translation for 'completed' is missing in 'nl-BE'

**Review by elpoelma — CHANGES_REQUESTED** [review]
Some minor comments regarding translations etc.

I did notice that the template comments are still included when copying/download a decision.

**Comment by piemonkey** [comment]
I'll fix the translation issues. The comment removal is in 2 parts. One is that I forgot to include the removal of comments from the preview, I'll fix that. In the copy/download though, I suspect the problem you're seeing is that due to the approach we took of marking the element for removal, the document must be created from a version of the template saved after the code change, otherwise it lacks the attribute to mark it for removal. Can you confirm whether the template you're looking at has this?

File: translations/en-us.yaml (hunk 2 above)
Comment (piemonkey) [reply translations/en-us.yaml:809]: Oops, I substituted 'completed' for 'copied'...

File: app/routes/meetings/download.js [hunk 3]
```diff
@@ -5,9 +5,10 @@ export default class MeetingsDownloadRoute extends Route {
   @service store;
   async model(params) {
     const zitting = await this.store.findRecord('zitting', params.id, {
-      include: 'bestuursorgaan,agendapunten.behandeling',
+      include:
+        'bestuursorgaan,agendapunten.behandeling,agendapunten.behandeling.document-container',
     });
-    zitting.agendapunten = (await zitting.agendapunten).sortBy('position');
+    zitting.agendapunten = [...(await zitting.agendapunten)].sortBy('position');
```
[the remark this answers is not in the document]
Comment (piemonkey) [reply app/routes/meetings/download.js:11]: True

File: translations/nl-BE.yaml [hunk 4]
```diff
@@ -801,7 +798,16 @@ download:
     agendapoint-title: Titel agendapunt
     download-html: Download HTML
     no-data-message: Geen agendapunten gevonden in zitting
+    copy-options-header: Kopieer naar klembord
+    copy-options: Bekijk kopieermogelijkehden
```
[the remark this answers is not in the document]
Comment (piemonkey) [reply translations/nl-BE.yaml:802]: Looks like a typo in the design, fixed.

**Comment by piemonkey** [comment]
I fixed the preview issue now. I've tested it with a locally published template from RB master and it removes the comment correctly.

**Pull request merged**